packages/investor-foxy/src/api/api.ts

```diff
@@ -121,7 +118,7 @@ export class FoxyApi {
    * @param amount
    */
   private normalizeAmount(amount: BigNumber) {
-    return this.web3.utils.toBN(amount.toFixed())
+    return ethers.BigNumber.from(amount.toFixed())
```

Comment:
This is only used internally so the ethers BigNumber flavor is correct - however the types are now wrong, this is assuming a `BigNumber.js` number. Update return type to reflect this

Reply:
Disregard me, this is *actually* fine: as was found previously, what ethers call `BigNumber` is in fact ethers own `BigNumber` flavor, which is properly typed as a return type here - still added an explicit return value